Library/Homebrew/cmd/switch.rb

```diff
@@ -19,7 +19,7 @@ def switch_args
         Symlink all of the specified <version> of <formula>'s installation into Homebrew's prefix.
       EOS
 
-      named 2
+      named_args number: 2
```

Comment:
Fair enough. I didn't bother because it's disabled and will be removed, but no harm. Same for `brew diy`